### PR TITLE
Add official centrality calibration for 14.6 GeV Run 19

### PR DESCRIPTION
**Pull request by nigmatkulov**
Official centrality calibration for 14.6 GeV Run19. Also added Makefile for standalone package compilation, which should work with ROOT6 (not with ROOT5 because of CINT).

### CODE REVIEW

**Comment by starsdong** [comment]
Hi Grigory, great to see this PR. I am about to send out the announcement for tomorrow's meeting. Will it be possible that you can give a presentation on this PR tomorrow? Thanks

**Comment by nigmatkulov** [comment]
> give a presentation on this PR tomorrow?

Hi Xin, are you talking about the S&C meeting? Do I need to prepare slides?

**Comment by starsdong** [comment]
Yes, Grigory. I think it will be up to you if you see necessary to show a few slides. Thanks

**Comment by nigmatkulov** [comment]
> Yes, Grigory. I think it will be up to you if you see necessary to show a few slides. Thanks

I think I can say a few words about this PR.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -75,7 +75,7 @@
 /StRoot/StPass0CalibMaker                          @genevb            @iraklic
 /StRoot/StPico*                                    @nigmatkulov       @marrbnl     @plexoos
 /StRoot/StPxl*                                     @starsdong         @GuannanXie
-/StRoot/StRefMultCorr                              @nigmatkulov
+/StRoot/StRefMultCorr                              @nigmatkulov       @zsweger
```
Comment (plexoos) [inline .github/CODEOWNERS:78]: An invitation to join star-bnl has been sent to this user previously but was not accepted.

Screen Shot 2022-08-17 at 12 30 03 PM

I suggest someone who knows this user communicates directly in order to add as a code owner with write privileges

Comment (nigmatkulov) [reply .github/CODEOWNERS:78]: How one can invite Zach to join the project? I did not find the proper button.

Comment (plexoos) [reply .github/CODEOWNERS:78]: The invitation has been sent and is pending. @zsweger needs to follow the link in the email.

**Review by marrbnl — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (plexoos) [reply .github/CODEOWNERS:78]: I see that Zach has accepted the invitation and became a maintainer of /StRoot/StRefMultCorr

**Review by plexoos — APPROVED** [review]

**Comment by starsdong** [comment]
Rongrong and Dmitri, shall we merge this PR so users can start to use it in their analysis? Thanks

**Comment by marrbnl** [comment]
OK with me. But I think Grigory wants to have Zach involved and go through the procedure.

**Comment by zsweger** [comment]
> OK with me. But I think Grigory wants to have Zach involved and go through the procedure.

Yep, I'm working on validating the changes and am writing some code to compare the expected performance of StRefMultCorr vs its actual performance. I had hoped to finish today, but it looks like it may take one more day.

**Comment by zsweger** [comment]
I edited a typo in the event-weight parameters that was giving some events a negative weight. I've tested the pileup rejection, Vz correction, and event weights on real data and they all now perform as expected. I sign off.

**Comment by starsdong** [comment]
Zach, we should merge this PR to the main branch. I am going to reopen it and merge it.

**Pull request merged**